### ISSUE TITLE
Xcode 9.3 beta 2 build issue

### ISSUE THREAD

**Issue by rae** [xmartlabs/Eureka#1412]
I'm not sure if you are tracking Xcode releases, but a possible Swift change caused this to come up today for me:

```[...]/Eureka/Source/Core/Section.swift:245:1: error: type 'Section' does not conform to protocol 'RangeReplaceableCollection'
extension Section: RangeReplaceableCollection {
^
[...]/Eureka/Source/Core/Section.swift:157:12: note: candidate has non-matching type '(Section) -> Void' [with SubSequence = Section.SubSequence]
    public init(_ initializer: (Section) -> Void) {
           ^
Swift.RangeReplaceableCollection:21:24: note: candidate has non-matching type 'S' [with SubSequence = Section.SubSequence]
    public convenience init<S>(_ elements: S) where S : Sequence, Self.Element == S.Element
                       ^
```

**Comment by scodino** [comment]
Same here

**Comment by rae** [comment]
I've tried to do some fixes, but I'm ending up with a new `init()` with no implementation which isn't right. You can see the attempts here: https://github.com/rae/Eureka/commits/fix/reid-fixes

**Comment by mtnbarreto** [comment]
Hi Guys! I will take a look shortly! probably next week!

**Comment by narlei** [comment]
This error here:
![screen shot 2018-02-14 at 15 21 30](https://user-images.githubusercontent.com/8092922/36218224-d864a230-119a-11e8-92c9-dfbb3cccc399.png)

**Comment by txaiwieser** [comment]
One issue seems to be related to Range protocols

**Comment by mats-claassen** [comment]
You can try the `swift4.1` branch which is compatible with the Xcode 9.3 beta 2

**Comment by arakweker** [comment]
Great! I've downloaded that branch... but how should I install it?

**Comment by mats-claassen** [comment]
If you are using Cocoapods:
```
pod 'Eureka', :git => 'https://github.com/xmartlabs/Eureka.git', :branch => 'swift4.1'
```

Carthage is similar.

If not just drag the Eureka.xcodeproj into your project

**Comment by arakweker** [comment]
Spot on. Eureka is working on Xcode 9.3 beta 2 now. Thanks.

**Comment by txaiwieser** [comment]
Thanks!

**Comment by che1404** [comment]
Awesome. Thanks! 🙌

**Comment by mtnbarreto** [comment]
Hey guys, we just released a new Eureka version that has Xcode 9.3 beta compatibility. Check it out here https://github.com/xmartlabs/Eureka/releases/tag/4.1.0
